contrib/platform/test/com/sun/jna/platform/win32/COM/ShellApplicationWindowsTest.java

```diff
@@ -0,0 +1,232 @@
+package com.sun.jna.platform.win32.COM;
+
+import java.util.Iterator;
+import java.util.NoSuchElementException;
+
+import com.sun.jna.Native;
+import com.sun.jna.NativeLibrary;
+import com.sun.jna.platform.win32.OleAuto;
+import com.sun.jna.platform.win32.Variant;
+import com.sun.jna.platform.win32.Variant.VARIANT;
+import com.sun.jna.platform.win32.WinDef.LONG;
+import com.sun.jna.platform.win32.WinDef.LONGLONG;
+
+import junit.framework.TestCase;
+
+public class ShellApplicationWindowsTest extends TestCase {
+
+	@Override
+	protected void setUp() throws Exception {
+		String[] commands = {"cmd", "/c", "start", "iexplore.exe","-nohome", "\"http://www.srh.noaa.gov/jax\""};
+		Runtime.getRuntime().exec(commands);
+		Thread.sleep(30000);
```

Comment:
I launched IE via Runtime.getRuntime().exec() because it's not the default browser on any of my systems - ShellExecute would open Chrome or Firefox on them.

If this will run only on boxes with IE as the default then I can adapt it accordingly.

As for the 30 second delay, that was the simplest way to give the page a chance to load. I'll change the target URL to "about:blank" - that should load instantly.